plugins/aws/ec2/unusedSecurityGroups.js

```diff
@@ -0,0 +1,67 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'Open All Ports Protocols',
+    category: 'EC2',
+    domain: 'Compute',
+    description: 'Determine if security group has all ports or protocols open to the public',
+    more_info: 'Security groups should be created on a per-service basis and avoid allowing all ports or protocols.',
+    link: 'http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/authorizing-access-to-an-instance.html',
+    recommended_action: 'Modify the security group to specify a specific port and protocol to allow.',
```

Comment:
```suggestion
    more_info: 'Keeping the number of security groups to a minimum makes the management easier and helps to avoid reaching the service limit.',
    link: 'https://docs.aws.amazon.com/vpc/latest/userguide/VPC_SecurityGroups.html',
    recommended_action: 'Remove security groups that are not being used.',
```